developer-docs-site/docs/tutorials/run-a-local-testnet.md

```diff
@@ -64,17 +68,17 @@ validator_1  | 	ChainId: TESTING
 ```
 
 This output contains information required for starting the Aptos CLI tool:
-* Aptos root key path - The root (also known as a mint or faucet) key controls the account that can mint. Available in the docker compose folder under `aptos_root_key`.
-* Waypoint - a verifiable checkpoint into the blockchain (available in the docker compose folder under waypoint.txt)
-* REST endpoint - `http://127.0.0.1:8080`.
-* ChainId - uniquely distinguishes this chain from other chains.
+* `Aptos root key path`: The root key (also known as the mint or faucet key) controls the account that can mint coins. Available in the docker compose folder under `aptos_root_key`.
+* `Waypoint`: A verifiable checkpoint of the blockchain (available in the docker compose folder under waypoint.txt)
+* `REST endpoint`: The endpoint for the REST service, e.g., `http://127.0.0.1:8080`.
+* `ChainId`: The chain id uniquely distinguishes this blockchain from other blockchains.
```

Comment:
this `network` from other `blockchain networks`.  Otherwise, it's sounding like it's differentiating between eth or btc.

Reply:
I would say Aptos-core compatible blockchain networks but agree.

Reply:
Went with @gregnazario's suggestion since it's shorter 😸